docs/general/introduction.rst

```diff
@@ -33,4 +31,32 @@ and viewing. SecureDrop Workstation combines all of those steps
 into one workflow on one machine: a Qubes computer that
 combines the *Journalist Workstation* and the *Secure Viewing Station*.
 
-For more information on SecureDrop Workstation, see our :doc:`faq <../journalist/faq>`.
+Who is behind SecureDrop Workstation?
+-------------------------------------
+SecureDrop and SecureDrop Workstation are open source projects of
+`Freedom of the Press Foundation (FPF) <https://freedom.press/>`_, a
+US-based nonprofit organization. You can support our work through
+`your donation <https://freedom.press/donate>`_ or by
+`contributing to SecureDrop development <https://developers.securedrop.org/en/latest/contributing.html>`_.
```

Comment:
A nit: maybe we can flip the order here and say "you can support our work by contributing to our projects [link], making a financial donation [link]," and (maybe?) adding "or following and sharing the work we do (newsletter/Mastodon link)"?.

I also prefer the "contributing to our projects" wording slightly over the "contributing to development," just to make sure the translators etc feel included :)

Reply:
I've flipped the order here and now use the more general "contributing" without specifying "development". I've not added the "follow/share news" just to keep it focused mainly on the core message of acknowledging the open source ecosystem we rely on.